### PR TITLE
Update AuthenticatesAndRegistersUsers.php

### PR DESCRIPTION
**Pull request by ssx**
Allow a logout path to be specified too

### CODE REVIEW

File: src/Illuminate/Foundation/Auth/AuthenticatesAndRegistersUsers.php [hunk 1]
```diff
@@ -106,8 +106,8 @@ protected function getFailedLoginMessage()
 	public function getLogout()
 	{
 		$this->auth->logout();
-
-		return redirect('/');
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesAndRegistersUsers.php]: Please remove trailing whitespace.

File: src/Illuminate/Foundation/Auth/AuthenticatesAndRegistersUsers.php [hunk 2]
```diff
@@ -106,8 +106,8 @@ protected function getFailedLoginMessage()
 	public function getLogout()
 	{
 		$this->auth->logout();
-
-		return redirect('/');
+		
+		return property_exists($this, 'logoutPath') ? $this->logoutPath : '/';
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesAndRegistersUsers.php]: is `isset` faster?

**Comment by taylorotwell** [comment]
logoutPath seems a little strange in wording. It's not the path to the logout screen. It's the path you want to go to after logging out.

**Comment by ssx** [comment]
Yeah was thinking that afterwards, maybe $logoutRedirect?

**Pull request closed without merging**